sklearn/manifold/_spectral_embedding.py

```diff
@@ -444,6 +466,26 @@ class SpectralEmbedding(BaseEstimator):
         to be installed. It can be faster on very large, sparse problems.
         If None, then ``'arpack'`` is used.
 
+    eigen_tol : float, default="auto"
+        Stopping criterion for eigendecomposition of the Laplacian matrix.
+        If `eigen_tol="auto"` then the passed tolerance will depend on the
+        `eigen_solver`:
+
+        - If `eigen_solver="arpack"`, then `eigen_tol=0.0`;
+        - If `eigen_solver="lobpcg"` or `eigen_solver="amg"`, then
+          `eigen_tol=None` which allows the underlying SciPy solver to
+          automaticaly resolve the value according to their heuristics.
```

Comment:
This can be updated to look like the other docstrings and link to `scipy.sparse.linalg.lobpcg`.